resolver/resolver.go

```diff
@@ -31,8 +31,9 @@ type Resolver struct {
 	rs               *records.RecordGenerator
 	rsLock           sync.RWMutex
 	rng              *rand.Rand
-	fwd              exchanger.Forwarder
 	generatorOptions []records.Option
+	zoneFwds         map[string]exchanger.Forwarder
```

Comment:
Yeah, I know what it it is, but I think it's valuable having a comment in the code explaining this.